libcxx/docs/Status/SpaceshipProjects.csv

```diff
@@ -83,7 +83,7 @@ Section,Description,Dependencies,Assignee,Complete
 "| `[string.view.synop] <https://wg21.link/string.view.synop>`_
 | `[string.view.comparison] <https://wg21.link/string.view.comparison>`_",| `basic_string_view <https://reviews.llvm.org/D130295>`_,None,Mark de Wever,|Complete|
 - `5.7 Clause 22: Containers library <https://wg21.link/p1614r2#clause-22-containers-library>`_,,,,
-| `[container.requirements.general] <https://wg21.link/container.requirements.general>`_,|,None,Unassigned,|Not Started|
+| `[container.requirements.general] <https://wg21.link/container.requirements.general>`_,|,None,Mark de Wever,|Complete|
```

Comment:
Can we consider the Spaceship paper as implemented or is it not there yet? Does it make sense to add a release note that Spaceship is now DONE (minus strong_order with long double)?

Reply:
Yes we can, I have a follow-up patch for that, which I upload when the CI is green.

That patch also updates the FTM so it feels better to do it in a separate patch.

Reply:
Yay! I was wondering if I should suggest to set the FTM and create an issue for the missing strong_equality case.

Reply:
I added a note to the status page and I have adding `long double` support on my personal TODO list. It seems the missing clang support makes progress so I have hope it can be worked on in the near future.